modules/ROOT/pages/ha-dr.adoc

```diff
@@ -0,0 +1,113 @@
+= High Availability and Disaster Recovery for Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Most high availability (HA) and disaster recovery (DR) strategies combine maximizing availability 
+within and across data centers, failover capability, and regular backups. The exact approach taken depends on your 
+organization's objectives for data loss tolerances and system recovery. 
+
+The following sections describe HA/DR strategies supported by Runtime Fabric.
+
+== Regions and Availability Zones with Cloud Providers
+
+Cloud providers, such as AWS and Azure, have data centers grouped by geographical regions. Within each region, multiple data centers are classified as Availability Zones. Redundancy can be achieved by creating a private network and using virtual machines across multiple availability zones. Detailed descriptions of regions and availability zones are provided by https://aws.amazon.com/about-aws/global-infrastructure/regions_az/[AWS] and https://docs.microsoft.com/en-us/azure/availability-zones/az-overview[Azure]. 
+
+The number of availability zones is dependent on the region selected for deployment. For high availability, select a region with at least 3 availability zones.
+
+For Runtime Fabric architecture specifications for development and production environments, refer to https://docs.mulesoft.com/runtime-fabric/1.3/architecture.
+
+== High Availability
+
+=== Availability Across Nodes
+By default, Runtime Fabric automatically detects and redeploys applications to available resources in the event of a node failure. To best leverage this functionality, Runtime Fabric should have at least one worker node’s worth of unused capacity. With a production configuration of Runtime Fabric, controller nodes can also tolerate a single node failure without impacting availability. 
+
+=== Availability Across Availability Zones
+High-speed, low latency private networks can span multiple availability zones within a region.
+
+Combining the availability across nodes with distributing nodes across different availability zones, Runtime Fabric can be configured to be highly available within a geographical region.
+
+Runtime Fabric is configured according to the type of environment:
+
+* Non-production environments require a minimum of two availability zones for high availability.
```

Comment:
Re-reading this, I think this is misleading given this configuration is not HA to begin with.
We could say for "improved availability".

Reply:
Updated this - thank you!

Reply:
Given this isn't actually HA, consider placing it last in the list of bullets.

Reply:
Moved this.